src/realm-settings/RealmSettingsSection.tsx

```diff
@@ -317,24 +340,20 @@ export const RealmSettingsSection = () => {
               data-testid="rs-localization-tab"
               title={<TabTitleText>{t("localization")}</TabTitleText>}
             >
-              {realm && (
-                <LocalizationTab
-                  key={key}
-                  refresh={refresh}
-                  save={save}
-                  reset={() => resetForm(realm)}
-                  realm={realm}
-                />
-              )}
+              <LocalizationTab
+                key={key}
+                refresh={refresh}
+                save={save}
+                reset={() => resetForm(realm)}
+                realm={realm!}
```

Comment:
If the aforementioned statement is changed there is no need to use a non-null assertion here.
```suggestion
                realm={realm}
```